metaflow/plugins/argo/argo_client.py

```diff
@@ -1,6 +1,4 @@
 import json
-import os
```

Comment:
Looks like my autoformatter went to town on some things... I'll revert those changes.